### DISCUSSION TITLE
Do people not use `Timer.one_shot` most of the time?

### DISCUSSION THREAD

**Discussion by feois** [godotengine/godot-proposals#7002] [Engine Core]
Having myself setting it as true 99% (actually 100%) of the time, I'm questioning the need to have it false by default so I'm asking if others have the same problem with me since I wanna make a proposal to change it to true by default.

**Comment by AThousandShips** [comment]
Changing the default break compatibility, and I don't think it's an incorrect default either, a lot of people probably use timers to trigger events periodically

**Comment by feois** [comment]
> a lot of people probably use timers to trigger events periodically

oh yeah, didn't think of that cuz I mainly use `Timer` for things like cooldown
> Changing the default break compatibility

good point
